### PR TITLE
fix connection when using RBAC on local cluster

### PR DESCRIPTION
**Pull request by xilabao**
Run `ALLOW_ANY_TOKEN=true ENABLE_RBAC=true hack/local-up-cluster.sh`  to start a local cluster 

When i run `./cluster/kubectl.sh get no --token=hi/system:masters` , I get a message: No resources found.

Add a user to kubeconfig can fix it.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/36146)

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

**Comment by dims** [comment]
Looks good to me 👍

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -366,7 +366,28 @@ function start_apiserver {
...
+      cluster: local-up-cluster
+      user: kubelet
+    name: service-to-apiserver
+current-context: service-to-apiserver
+users:
+- name: kubelet
+  user:
+      token: admin/system:masters
+EOF
+  else
```
Comment (liggitt) [inline hack/local-up-cluster.sh]: what about something like #36134? don't more things than just the master use this kubeconfig? the way we start local-up-cluster with separately secured components could probably use some rethinking in 1.6

**Comment by xilabao** [comment]
use the new command can fix the problem
`ALLOW_ANY_TOKEN=true KUBECONFIG_TOKEN=admin/system:masters ENABLE_RBAC=true hack/local-up-cluster.sh`
doc updated #36041

**Comment by xilabao** [comment]
closed

**Pull request closed without merging**